rhizome/host/lib/vm_setup.rb

```diff
@@ -482,15 +488,17 @@ def write_user_data(unix_user, public_keys, swap_size_bytes)
     ssh_authorized_keys:
 #{public_keys.map { "      - #{yq(_1)}" }.join("\n")}
 
+packages:
+  - nftables
```

Comment:
https://cloudinit.readthedocs.io/en/latest/reference/examples.html#install-arbitrary-packages
> # if packages are specified, then package_update will be set to true
So it will increase provisioning times for runners 🙃